Detectors/ITSMFT/ITS/reconstruction/src/CookedTracker.cxx

```diff
@@ -397,11 +386,10 @@ void CookedTracker::trackSeeds(std::vector<TrackITS>& seeds)
     Double_t x = track.getX();
     Double_t y = track.getY();
     Double_t phi = track.getAlpha() + TMath::ATan2(y, x);
-    const Float_t pi2 = 2. * TMath::Pi();
     if (phi < 0.)
-      phi += pi2;
-    else if (phi >= pi2)
-      phi -= pi2;
+      phi += k2PI;
+    else if (phi >= k2PI)
+      phi -= k2PI;
```

Comment:
Why do you still use doubles here? 
Just for the info: the file ``O2/Common/MathUtils/include/MathUtils/Utils.h`` provides a set of methods for dealing with angle wrapping, including the ``BringTo02Pi``

Reply:
@shahor02 Thank you, Ruben, for the suggestions.
Actually, I have found one more place like that.  All are fixed now.